### ISSUE TITLE
Object Detection Training Stops With "^C" Need Help

### ISSUE THREAD

**Issue by keremnymn** [tensorflow/models#9141]
Hello, I'm trying to train EfficientDet on my custom dataset. In spite of the EfficientDet backbone (D0-D1-D2-D3), Google Colab just kicks me out giving an error like this:

`W0823 08:47:59.941824 140512077502208 optimizer_v2.py:1275] Gradients do not exist for variables ['top_bn/gamma:0', 'top_bn/beta:0'] when minimizing the loss.
WARNING:tensorflow:From /usr/local/lib/python3.6/dist-packages/tensorflow/python/util/deprecation.py:574: calling map_fn_v2 (from tensorflow.python.ops.map_fn) with dtype is deprecated and will be removed in a future version.
Instructions for updating:
Use fn_output_signature instead
W0823 08:48:01.099456 140512077502208 deprecation.py:506] From /usr/local/lib/python3.6/dist-packages/tensorflow/python/util/deprecation.py:574: calling map_fn_v2 (from tensorflow.python.ops.map_fn) with dtype is deprecated and will be removed in a future version.
Instructions for updating:
Use fn_output_signature instead
WARNING:tensorflow:Gradients do not exist for variables ['top_bn/gamma:0', 'top_bn/beta:0'] when minimizing the loss.
W0823 08:48:24.028264 140512077502208 optimizer_v2.py:1275] Gradients do not exist for variables ['top_bn/gamma:0', 'top_bn/beta:0'] when minimizing the loss.
tcmalloc: large alloc 1610612736 bytes == 0x28d7cc000 @  0x7fcbe4652b6b 0x7fcbe4672379 0x7fcbc8c90207 0x7fcbba1bec4f 0x7fcbba24748b 0x7fcbba0b9d06 0x7fcbba0babcc 0x7fcbba0bae53 0x7fcbc4ed9189 0x7fcbba33c57f 0x7fcbba331605 0x7fcbba3ef591 0x7fcbba3ec2a3 0x7fcbba3db3e5 0x7fcbe40346db 0x7fcbe436da3f
^C`

I've tried batch sizes 64 and 128, no luck. All the images in my dataset is 896x896. I don't know where I'm doing the mistake.

Can anybody please help?

**Comment by albertoisorna** [comment]
Is there any solutions to this??

**Comment by albertoisorna** [comment]
Reducing the batch size to 4 in Google Colab worked for me 👍

**Comment by kilarinikhil** [comment]
I had the same problem decreasing image dim and batch size in config file resolved the issue. I think this is because not enough ram being available while training

**Comment by vmishra2019** [comment]
I had the same problem decreasing batch size in config (made it 8) file resolved the issue.

**Comment by Seh83** [comment]
Try experimenting with batch_size in pipeline.config file... a smaller the batch size of 4 or less will solve the issue.

**Comment by Ahmadmustapha1** [comment]
Thank you all, batch_size was 8 and it got stuck. By reducing to 4, it runs succesfully.

**Comment by AtosheIslamSumaya** [comment]
> I had the same problem decreasing image dim and batch size in config file resolved the issue. I think this is because not enough ram being available while training

how to decrease the image dim? i am not seeing any option
